### ISSUE TITLE
why "weights" in features?

### ISSUE THREAD

**Issue by xilinniao123** [thunlp/XQA#2]
hi:
  thanks for sharing! one question about weights,
it's  usage is in compute_loss(), and it only effect end_logits computation!
in the generation of weights ,"[1] * len(start_positions)" , when 1 is represents start begins.
so, my problem is why use "weights "  to compute end logits loss?

**Comment by alphaf52** [comment]
Sorry for late reply. This file is modified from the original version of DocumentQA, and I did not change this particular line. I will try to figure that out.
